docker/install/ubuntu1804_install_llvm.sh

```diff
@@ -36,10 +36,15 @@ echo deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-11 main\
 echo deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic-11 main\
      >> /etc/apt/sources.list.d/llvm.list
 
+echo deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-12 main\
+     >> /etc/apt/sources.list.d/llvm.list
+echo deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic-12 main\
+     >> /etc/apt/sources.list.d/llvm.list
+
 echo deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main\
      >> /etc/apt/sources.list.d/llvm.list
 echo deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic main\
      >> /etc/apt/sources.list.d/llvm.list
 
 wget -q -O - http://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
-apt-get update && apt-get install -y llvm-9 llvm-10 llvm-11 clang-9 libclang-9-dev clang-10 libclang-10-dev clang-11 libclang-11-dev
+apt-get update && apt-get install -y llvm-9 llvm-10 llvm-11 llvm-12 clang-9 libclang-9-dev clang-10 libclang-10-dev clang-11 libclang-11-dev clang-12 libclang-12-dev
```

Comment:
Given we usually maintain the last 3 version, I think we can remove version 9 in the same PR, keeping only 10, 11 and 12?

Also cc @tqchen @jroesch for reviews

Reply:
Let us first keep all, we might need to update testcase first to change the llvm-config before we can drop 9

Reply:
Understood.